### PR TITLE
A few change propositions

### PR DESCRIPTION
**Pull request by krystianbialy**
Hi, 
i'm submitting 2 commits.
1. The first one is only a proposition of local fonts loading. 
2. Second commit is a text fixing in a footer.
footer text fix

### CODE REVIEW

**Comment by mlazowik** [comment]
Thanks for the PR!

Could you replace all fonts with woff2 versions? ttf is not compressed at all. Google serves woff2 and we're getting ~10KiB per font style, here it's ~100-200KiB.

Also in general I'd say that it makes more sense to submit separate things as separate pull requests, makes it easier to iterate on them and accept them.

File: src/fonts/fonts.css [hunk 1]
```diff
@@ -1,9 +1,27 @@
...
 @font-face {
-    font-family: IBM Plex Sans;
-    src: url(./IBMPlexSans-Bold.ttf);
+    font-family: 'IBM Plex Sans';
+    font-style: normal;
+    font-weight: 700;
+    src: url('./IBMPlexSans-Bold/IBMPlexSans-Bold.eot'); /* IE9 Compat Modes */
+    src: local('IBM Plex Sans Bold'), local('IBMPlexSans-Bold'),
+         url('./IBMPlexSans-Bold/IBMPlexSans-Bold.eot?#iefix') format('embedded-opentype'), /* IE6-IE8 */
+         url('./IBMPlexSans-Bold/IBMPlexSans-Bold.woff2') format('woff2'), /* Super Modern Browsers */
```
Comment (mlazowik) [inline src/fonts/fonts.css:9]: woff2 is supported by everything apart from IE 11 and opera mini

File: src/fonts/fonts.css [hunk 2]
```diff
@@ -1,9 +1,27 @@
...
+         url('./IBMPlexSans-Bold/IBMPlexSans-Bold.woff') format('woff'), /* Modern Browsers */
```
Comment (mlazowik) [inline src/fonts/fonts.css:10]: woff is supported by everything apart from IE 8 (pretty sure our app does not work there either way) and opera mini

**Comment by krystianbialy** [comment]
I left only the woff2 format.

**Comment by mlazowik** [comment]
Thanks! @micorix IMO looks great now.

**Pull request merged**